docs/issues/71851376d2af5f56955bf728d321d90b.md

```
@@ -1,30 +1,79 @@
 Severity: medium
+Priority: medium (Low volume, edge case, but ANR)
 
 [View in Firebase Console](https://console.firebase.google.com/project/d-print-cost-calculator-cf650/crashlytics/app/android:com.threed_print_calculator/issues/71851376d2af5f56955bf728d321d90b)
 
 ## Summary
-ANR triggered by thread waiting for a binder transaction
+ANR triggered by thread waiting for a binder transaction. The main thread is blocked in `libc.so __ioctl` while calling `IWindowSession.getInTouchMode` during `ViewRootImpl.performTraversals`.
 
 ## Stack trace
-(Refer to Firebase Console link above for full trace)
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)